src/Stoa.ts

```diff
@@ -173,39 +175,44 @@ class Stoa extends WebService {
+        this.app.get("/transaction/pending/:hash", isBlackList, this.getTransactionPending.bind(this));
+        this.app.get("/transaction/:hash", isBlackList, this.getTransaction.bind(this));
+        this.app.get("/utxo/:address", isBlackList, this.getUTXO.bind(this));
+        this.app.post("/utxos", isBlackList, this.getUTXOs.bind(this));
+        this.app.get("/transaction/status/:hash", isBlackList, this.getTransactionStatus.bind(this));
+        this.app.get("/transaction/fees/:tx_size", isBlackList, this.getTransactionFees.bind(this));
+        this.app.get("/wallet/transactions/history/:address", isBlackList, this.getWalletTransactionsHistory.bind(this));
+        this.app.get("/wallet/transaction/overview/:hash", isBlackList, this.getWalletTransactionOverview.bind(this));
+        this.app.get("/wallet/transactions/pending/:address", isBlackList, this.getWalletTransactionsPending.bind(this));
+        this.app.get("/wallet/blocks/header", isBlackList, this.getWalletBlocksHeader.bind(this));
+        this.app.get("/latest-blocks", isBlackList, this.getLatestBlocks.bind(this));
+        this.app.get("/latest-transactions", isBlackList, this.getLatestTransactions.bind(this));
+        this.app.get("/block-summary", isBlackList, this.getBlockSummary.bind(this));
+        this.app.get("/block-enrollments", isBlackList, this.getBlockEnrollments.bind(this));
+        this.app.get("/block-transactions", isBlackList, this.getBlockTransactions.bind(this));
+        this.app.get("/boa-stats", isBlackList, this.getBOAStats.bind(this));
+        this.app.get("/spv/:hash", isBlackList, this.verifyPayment.bind(this));
+        this.app.get("/coinmarketcap", isBlackList, this.getCoinMarketCap.bind(this));
+        this.app.get("/coinmarketchart", isBlackList, this.getBoaPriceChart.bind(this));
         this.app.post("/block_externalized", this.postBlock.bind(this));
         this.app.post("/block_header_updated", this.postBlockHeaderUpdate.bind(this));
         this.app.post("/preimage_received", this.putPreImage.bind(this));
         this.app.post("/transaction_received", this.putTransaction.bind(this));
-        this.app.get("/holders", this.getBoaHolders.bind(this));
-        this.app.get("/holder_balance_history/:address", this.getHolderBalanceHistory.bind(this));
-        this.app.get("/holder/:address", this.getBoaHolder.bind(this));
-        this.app.get("/average_fee_chart", this.averageFeeChart.bind(this));
-        this.app.get("/search/hash/:hash", this.searchHash.bind(this));
+        this.app.get("/holders", isBlackList, this.getBoaHolders.bind(this));
+        this.app.get("/holder_balance_history/:address", isBlackList, this.getHolderBalanceHistory.bind(this));
+        this.app.get("/holder/:address", isBlackList, this.getBoaHolder.bind(this));
+        this.app.get("/average_fee_chart", isBlackList, this.averageFeeChart.bind(this));
+        this.app.get("/search/hash/:hash", isBlackList, this.searchHash.bind(this));
```

Comment:
There is an endpoint that has been added.
v0.x.x Rebase is required.

Reply:
resolved